recipes/ckzg/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ckzg" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ckzg-{{ version }}.tar.gz
+  sha256: 4295acc380f8d42ebea4a4a0a68c424a322bb335a33bad05c72ead8cbb28d118
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('clang') }}  # [linux]
```

Comment:
A cleaner approach for this is described in https://conda-forge.org/news/2024/04/30/clang-everywhere/ Havig both `compiler('c')` and `compiler('clang')` can lead to issues. Putting clang into cbc.yaml should prevent this.

Reply:
I tried that but it fails unfortunately. This might be because the build script explicitly uses gcc for windows. https://github.com/step21/c-kzg-4844/blob/main/src/Makefile

Reply:
Best is to limit it with `  # [linux]` on all lines of the cbc.yaml

Reply:
Somehow this is not working. Error from OSX `  /var/folders/n8/w901w1rn4ldg6yskstndsj200000gn/T/assembly-f95753.s:6555:1: error: invalid CFI advance_loc expression
  .cfi_adjust_cfa_offset 8`
Windows build just tries to apply clang options to gcc, which fails ofc.

Reply:
@xhochy  Please advise if I should fix/remove the build script from upstream to enable clang for all (assuming it works) or revert to the version that worked using the built-in build script, but uses gcc for windows.